src/include/ndpi_win32.h

```diff
@@ -48,6 +51,17 @@
 
 #define	IPVERSION	4 /* on *nix it is defined in netinet/ip.h */ 
 
+#ifndef MIN
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+#endif
+
+#ifndef IPPROTO_SCTP
+#define IPPROTO_SCTP 132
+#endif
+
+#undef gettimeofday
+#define gettimeofday mingw_gettimeofday
```

Comment:
Not good for clang-cl:
```
ElasticSearch.cpp(140,7): error: use of undeclared identifier 'mingw_gettimeofday'; did you mean 'gettimeofday'?
      gettimeofday(&tv, NULL);
      ^~~~~~~~~~~~
      gettimeofday
../nDPI/src/include/ndpi_win32.h(63,22): note: expanded from macro 'gettimeofday'
#define gettimeofday mingw_gettimeofday
                     ^
../include/ntop_win32.h(93,12): note: 'gettimeofday' declared here
extern int gettimeofday(struct timeval * tp, struct timezone * tzp);
           ^
```

Reply:
Whoops. Totally forgot to test this PR with clang-cl